doc/mkdocs/docs/api/macros/json_legacy_comparison.md

```diff
@@ -0,0 +1,56 @@
+# JSON_LEGACY_COMPARISON
+
+```cpp
+#define JSON_LEGACY_COMPARISON /* value */
+```
+
+This macro enables the (incorrect) legacy comparison behavior of discarded JSON values.
+Possible values are `1` to enable or `0` to disable (default).
+
+When enabled, comparisons involving at least one discarded JSON value yield results as follows:
+
+| **Operator** | **Result**    |
+|--------------|---------------|
+| `==`         | `#!cpp false` |
+| `!=`         | `#!cpp true`  |
+| `<`          | `#!cpp false` |
+| `<=`         | `#!cpp true`  |
+| `>=`         | `#!cpp true`  |
+| `>`          | `#!cpp false` |
+
+Otherwise, comparisons involving at least one discarded JSON value always yield `#!cpp false`.
+
+## Default definition
+
+The default value is `0`.
+
+```cpp
+#define JSON_LEGACY_COMPARISON 0
+```
+
+When the macro is not defined, the library will define it to its default value.
+
+## Notes
+
+!!! warning "Inconsistent behavior in C++20 and beyond"
+
+    When targeting C++20 or above, enabling the legacy comparison behavior is _strongly_
+    discouraged.
+
+      - The `<=>` operator will always give the correct result (`#!cpp false`).
+      - Overloads for the equality and relational operators emulate the legacy behavior.
+
+    Code outside your control may use either the 3-way comparison or the equality and
```

Comment:
Maybe a bit nitpicky, but mixing "<=>" (above) and "3-way comparison" (here) may confuse readers new to C++20.